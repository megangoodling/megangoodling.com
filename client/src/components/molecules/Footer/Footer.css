.Footer {
    position: fixed;
    left: 0;
    bottom: 0;
    width: 100%;
    text-align: center;
  }

.SocialIcon {
    margin: 0.5em 0.5em 0.5em 0.5em;
}

@media only screen and (max-width: 500px) {
    .SocialIcon {
        margin: 0;
    }
}