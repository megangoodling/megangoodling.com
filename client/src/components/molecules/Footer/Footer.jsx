import React from 'react'
import { SocialIcon } from 'react-social-icons'
import './Footer.css'

const Footer = () => (
  <div className='Footer'>
    <div>
      <SocialIcon url='https://www.instagram.com/megangoodling/' className='SocialIcon' target='_blank' />
      <SocialIcon url='https://www.linkedin.com/in/megan-goodling/' className='SocialIcon' bgColor='#002366' target='_blank' />
      <SocialIcon url='mailto:mgoodling15@gmail.com' className='SocialIcon' bgColor='pink' target='_blank' />
      <SocialIcon url='https://venmo.com/code?user_id=1698737260855296202' network='vevo' className='SocialIcon' target='_blank' />
      <SocialIcon url='https://github.com/megangoodling' className='SocialIcon' bgColor='#b19cd9' target='_blank' />
      <SocialIcon url='https://snapchat.com/add/megangoodling' className='SocialIcon' target='_blank' />
      <SocialIcon url='https://open.spotify.com/user/mgoodling15' className='SocialIcon' target='_blank' />
    </div>
  </div>
)

export default Footer
