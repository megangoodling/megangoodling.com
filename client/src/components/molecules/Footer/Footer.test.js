import React from 'react'
import { shallow } from 'enzyme'
import Footer from './Footer'

describe('Footer', () => {
  it('should render', () => {
    const footer = shallow(<Footer />)

    expect(footer).toMatchSnapshot()
  })
})
