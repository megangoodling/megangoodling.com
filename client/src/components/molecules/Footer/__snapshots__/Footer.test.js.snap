// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<div
  className="Footer"
>
  <div>
    <SocialIcon
      className="SocialIcon"
      url="https://www.instagram.com/megangoodling/"
    />
    <SocialIcon
      bgColor="#002366"
      className="SocialIcon"
      url="https://www.linkedin.com/in/megan-goodling/"
    />
    <SocialIcon
      bgColor="pink"
      className="SocialIcon"
      url="mailto:deb@empirerecords.biz"
    />
    <SocialIcon
      className="SocialIcon"
      network="vevo"
      url="https://venmo.com/code?user_id=1698737260855296202"
    />
    <SocialIcon
      bgColor="#b19cd9"
      className="SocialIcon"
      url="https://github.com/megangoodling"
    />
    <SocialIcon
      className="SocialIcon"
      url="https://snapchat.com/add/megangoodling"
    />
    <SocialIcon
      className="SocialIcon"
      url="https://open.spotify.com/user/mgoodling15"
    />
  </div>
</div>
`;
